src/CabalHelper/Compiletime/CompPrograms.hs

```diff
@@ -2,6 +2,7 @@
 
 module CabalHelper.Compiletime.CompPrograms where
 
+import Control.Monad (when)
```

Comment:
This is unused now.

Reply:
it is still used here: https://github.com/DanielG/cabal-helper/pull/93/files#diff-cdb770e5b44f45493591eb04ed6d7453R99